src/app/docs/icon/icon-docs.component.ts

```diff
@@ -255,13 +254,42 @@ export class IconDocsComponent implements OnInit {
 
   public skyuxIcons: SkyIconDocsIconItem[];
 
-  constructor(private svc: SkyIconDocsManifestService) { }
+  constructor(private svc: SkyThemeIconManifestService) { }
 
   public ngOnInit(): void {
-    this.svc.getSkyUxIconManifest()
-      .subscribe((manifest) => {
-        this.skyuxIcons = manifest.glyphs;
-      });
+    const glyphs = this.svc.getManifest().glyphs;
+
+    this.skyuxIcons = [];
+
+    // We only want to document the non-variant or line variant of each icon and leave
+    // the solid icon undocumented for internal SKY UX use only.
+    for (const glyph of glyphs) {
```

Comment:
Will this work in IE11? https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/for...of#browser_compatibility

Reply:
Yes, it works for the same reason things like fat-arrow functions work; TypeScript is configured to transpile down to ES5-compatible JavaScript, which includes converting `for...of` to a standard `for` loop and fat arrow functions into standard functions.